pkgs/development/python-modules/jupyterhub/default.nix

```diff
@@ -51,7 +51,7 @@ in
 
 buildPythonPackage rec {
   pname = "jupyterhub";
-  version = "0.9.4";
+  version = "1.0.0";
```

Comment:
Without the update of `sha256` this has no effect. This would build on your machine because the download is cached for the current hash (which is a tarball of 0.9.4).